src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="nav__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="navBox">
+        <ul class="liBox">
+          <li>
+            <a  href="index.html" class="is-active"><span class="is-active">Apple</span></a>
+          </li>
+          <li>
+            <a  href="index.html"><span>Samsung</span></a>
+          </li>
+          <li>
+            <a  href="index.html"><span>Smartphones</span></a>
+          </li>
+          <li>
+            <a  href="index.html" data-qa="hover"><span>Laptops & Computers</span></a>
+          </li>
+          <li>
+            <a  href="index.html"><span>Gadgets</span></a>
+          </li>
+          <li>
+            <a  href="index.html"><span>Tablets</span></a>
+          </li>
```

Comment:
Add classes everywhere for tag li and a
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="index.html">Tablets</a>
          </li>
```